generators/typescript/express/express-endpoint-type-schemas-generator/src/GeneratedExpressEndpointTypeSchemasImpl.ts

```diff
@@ -173,6 +195,17 @@ export class GeneratedExpressEndpointTypeSchemasImpl implements GeneratedExpress
                 if (this.generatedResponseSchema == null) {
                     throw new Error("No response schema was generated");
                 }
+                if (this.allowExtraFields) {
+                    return this.generatedResponseSchema
+                        .getReferenceToZurgSchema(context)
+                        .jsonOrThrow(referenceToParsedResponse, {
+                            unrecognizedObjectKeys: "passthrough",
+                            allowUnrecognizedEnumValues: true,
+                            allowUnrecognizedUnionMembers: true,
+                            skipValidation: true,
```

Comment:
should we be skipping validation here?

Reply:
No, good catch - updated with a `getSchemaOptions` function (as discussed).